documentation/api/guides/https.md

```diff
@@ -0,0 +1,48 @@
+# HTTPS
+
+To access the API over HTTPS, we need to proxy to it using NGINX. Note that this
+proxy is different from the API NGINX image that is specifically aimed at
+serving static files.
```

Comment:
Just a small clarification as we do not use that image outside of live environments.

```suggestion
To access the API over HTTPS, we need to proxy to it using NGINX. Note that this
proxy is different from the API NGINX image that is specifically aimed at
serving static files in live environments.
```